src/segment-loader.js

```diff
@@ -717,6 +717,43 @@ export default class SegmentLoader extends videojs.EventTarget {
     };
   }
 
+  abortRequestEarly_(stats) {
+    const firstByteReceived = Date.now() - stats.firstByteReceived;
+    // Since bandwidth values reported by progress events stabilize as time goes on,
+    // we adjust bandwidth by a conservative ammount, increasing as more time has passed
+    // maxing at 0.8 to prevent aborting a request we actually do have enough bandwidth
+    // for.
+    const playlistBandwidthAdjustment = Math.min(0.8, firstByteReceived / 5000);
```

Comment:
Well, I feel like this adjustment number is doing 2 things. 
1. is accounting for bandwidth ramp up, as time goes on, bandwidth will have ramped up and become more stable, so we can use a value closer to the real bandwidth of the playlist. This is also why below we make sure its been about 1 second since first byte seen. I've run some (granted not very exhaustive) experiments that seemed to show around 1 second is when bandwidth from the progress events stabilizes.
2. is that increasing the adjustment as time goes on, allows us to be more likely to emergency abort when the request is taking too long. I like the idea of using segment duration here because 5 seconds may not be too long for 10 second segments, but for 2 second segments it is definitely way too long, and we probably should've been using the full `08.` well before 4 seconds since request